ui/common/css/component/_user-link.scss

```diff
@@ -40,6 +40,9 @@
     }
   }
 }
-a.user-link:hover {
-  color: $c-link;
+a.user-link {
+  @include transition();
+  &:hover {
+    color: $c-link;
+  }
```

Comment:
We don't want site-wide transitions like that. Not all things need to be animated.